addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/WebserviceHandler.java

```diff
@@ -66,7 +69,19 @@ public WebserviceHandler(Bridge bridge) {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        // no code needed.
+        try {
+            if (channelUID.getId().contentEquals(CHANNEL_HOLIDAY)) {
+                logger.debug("HandleCommand recieved: {}", channelUID.getId());
+                if (command.toString() == "OFF") {
+                    isHoliday = false;
+                } else {
+                    isHoliday = true;
+                }
+            }
+        } catch (Exception e) {
```

Comment:
I'm not sure what kind of error to catch here?
The routine comes with two arguments, the channelUID and the command.
First the channelUID is checked wether it equals an enum, IMHO there is no error possible.
Using your shortened suggestion above all possible outcomes are covered. could the Try..Catch be removed in toto, then?